<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>350</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:0.267, stop:0 rgba(140, 255, 0, 138), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>30</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 11pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Привет! меня зовут Айтигеник. Я помогу тебе выбрать направления в нашей школе. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>360</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36ff00;
border-radius: 15px

</string>
     </property>
     <property name="text">
      <string>Подбор направления </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="recources.qrc">:/itgenik.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>289</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Заниматься творчеством 
(рисовать, лепить, фантазировать)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 11pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;    Подскажи, пожалуйста, &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;что тебе нравится больше?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>321</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Мечтаю программировать, 
изучать алгоритмы, запускать ракеты в 
космос!</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>321</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Люблю творчество, и программирование!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>360</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36ff00;
border-radius: 15px</string>
     </property>
     <property name="text">
      <string>Далее </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 11pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Итак, творческая личность. Теперь по конкретнее:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>297</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Xочу рисовать, редактировать фото.</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>277</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Хочу моделировать здания, предметы 
интерьера, конструировать фигурки и 
печатать их на 3D - принтере </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>193</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Хочу создавать музыку</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>360</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36ff00;
border-radius: 15px</string>
     </property>
     <property name="text">
      <string>Далее</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 4</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 11pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;У тебя серьёзные намерения, мой друг! Теперь поконкретнее:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>297</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Хочу делать сайты, веб-приложения.</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>297</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Хочу заниматься обработкой данных, 
алгоритмами и писать серьезный код</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>257</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Хочу создавать приложения для 
смартфонов!</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>321</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Я пока не програмировал(-а) и не знаю 
чего хочу</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>185</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 10pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>Хочу создавать игры!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>360</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #36ff00;
border-radius: 15px</string>
     </property>
     <property name="text">
      <string>Далее</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Tab 5</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font: 11pt &quot;Lucida Console&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Тадаааа!... Вот что тебе предлагает изучить Айтигеник: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
